core/field_textinput.js

```diff
@@ -251,7 +256,7 @@ Blockly.FieldTextInput.prototype.showInlineEditor_ = function(quietInput) {
 Blockly.FieldTextInput.prototype.widgetCreate_ = function() {
   var div = Blockly.WidgetDiv.DIV;
 
-  var htmlInput = document.createElement('input');
+  var htmlInput = /** @type {HTMLInputElement} */ (document.createElement('input'));
```

Comment:
This is just a me-knowledge question, unrelated to your PR: Is this to help the compiler? I see these type tags sometimes and I don't know what they're for.

Reply:
So, this is a type cast. ``document.createElement`` always returns a type of ``Element``. But ``htmlInput`` as indicated above is of type ``HtmlInputElement`` so we cast ``Element`` to ``HtmlInputElement`` to please the compiler. (This is because we KNOW that when we call ``createElement`` with ``'input'`` we are going to get an input element (ie: an element of type ``HtmlInputElement``).
Another thing I could've done here is make the type of ``htmlInput`` as jut the generic ``Element``. The problem with that is it's not very specific. The compiler will complain about doing things like ``htmlInput.style.XYZ`` because ``Element`` doesn't have a ``style`` property, but ``HtmlElement`` / ``HtmlInputElement`` do.
Does that make sense?

Try to be more specific with types if you know what the type is going to be, and *allegedly* the compiler will help you find mistakes!